mm2src/mm2_main/src/mm2.rs

```diff
@@ -274,80 +274,64 @@ pub async fn lp_main(params: LpMainParams, ctx_cb: &dyn Fn(u32)) -> Result<(), S
-            "  passphrase *   ..  Wallet seed.\n"
-            "                     Compressed WIFs and hexadecimal ECDSA keys (prefixed with 0x) are also accepted.\n"
-            "  panic          ..  Simulate a panic to see if backtrace works.\n"
-            // cf. https://github.com/atomiclabs/hyperdex/pull/563/commits/6d17c0c994693b768e30130855c679a7849a2b27
-            "  rpccors        ..  Access-Control-Allow-Origin header value to be used in all the RPC responses.\n"
-            "                     Default is currently 'http://localhost:3000'\n"
-            "  rpcip          ..  IP address to bind to for RPC server. Overrides the 127.0.0.1 default\n"
-            "  rpc_password   ..  RPC password used to authorize non-public RPC calls\n"
-            "                     MM generates password from passphrase if this field is not set\n"
-            "  rpc_local_only ..  MM forbids some RPC requests from not loopback (localhost) IPs as additional security measure.\n"
-            "                     Defaults to `true`, set `false` to disable. `Use with caution`.\n"
-            "  rpcport        ..  If > 1000 overrides the 7783 default.\n"
-            "  i_am_seed      ..  Activate the seed node mode (acting as a relay for mm2 clients).\n"
-            "                     Defaults to `false`.\n"
-            "  seednodes      ..  Seednode IPs that node will use.\n"
-            "                     At least one seed IP must be present if the node is not a seed itself.\n"
-            "  stderr         ..  Print a message to stderr and exit.\n"
-            "  userhome       ..  System home directory of a user ('/root' by default).\n"
-            "  wif            ..  `1` to add WIFs to the information we provide about a coin.\n"
-            "\n"
-            "Environment variables:\n"
-            "\n"
-            // Per-process (we might have several MM instances running but they will share the same log).
-            "  MM_CONF_PATH   ..  File path. MM2 will try to load the JSON configuration from this file.\n"
-            "                     File must contain valid json with structure mentioned above.\n"
-            "                     Defaults to `MM2.json`\n"
-            "  MM_COINS_PATH  ..  File path. MM2 will try to load coins data from this file.\n"
-            "                     File must contain valid json.\n"
-            "                     Recommended: https://github.com/jl777/coins/blob/master/coins.\n"
-            "                     Defaults to `coins`.\n"
-            "  MM_LOG         ..  File path. Must end with '.log'. MM will log to this file.\n"
-            "\n"
-            // Generated from https://github.com/KomodoPlatform/developer-docs/tree/sidd.
-            // (SHossain, siddhartha-crypto).
-            "See also the online documentation at\n"
-            "https://developers.atomicdex.io\n"
-        )
+    const HELP_MSG: &str = r#"Command-line options.
```

Comment:
Is `#[allow(dead_code)]` required above?

Reply:
It doesn't seem to be needed anymore. Removed it